spec/funnel_cake_index_spec.rb

```diff
@@ -0,0 +1,9 @@
+RSpec.describe FunnelCakeIndex do
+  it "has a version number" do
+    expect(FunnelCakeIndex::VERSION).not_to be nil
+  end
+
+  it "does something useful" do
```

Comment:
I think even a super minimal test  would be great to ensure that the config loads.

Reply:
Added Specs